src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -218,8 +218,10 @@ private FunctionDescr Scan()
             if (_info == null)
             {
                 var t = GetType();
-                var suffix = "Function";
-                var name = t.Name.Substring(0, t.Name.Length - suffix.Length);
+                var suffix = "Function";
+
+                //                 var name = t.Name.Substring(0, t.Name.Length - suffix.Length);  and replace Program.cs, Decimal TODO before PR
```

Comment:
>Decimal TODO before PR [](http://example.com/codeflow?start=123&length=22)

Before PR?

Reply:
Oops, this is destined for a different PR.  I'll remove.